<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Agregar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Particula</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="12" column="3" colspan="2">
            <widget class="QLineEdit" name="Green_lineEdit"/>
           </item>
           <item row="1" column="3" colspan="2">
            <widget class="QLineEdit" name="Origen_Y_lineEdit"/>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="QLabel" name="DestinoX">
             <property name="text">
              <string>Destino X: (0 - 500) </string>
             </property>
            </widget>
           </item>
           <item row="2" column="3" colspan="2">
            <widget class="QLineEdit" name="DestinoX_lineEdit"/>
           </item>
           <item row="14" column="1">
            <widget class="QLabel" name="Blue">
             <property name="text">
              <string>Blue: (0 - 255)</string>
             </property>
            </widget>
           </item>
           <item row="8" column="3" colspan="2">
            <widget class="QLineEdit" name="Red_lineEdit"/>
           </item>
           <item row="12" column="1">
            <widget class="QLabel" name="Green">
             <property name="text">
              <string>Green: (0 - 255)</string>
             </property>
            </widget>
           </item>
           <item row="4" column="3" colspan="2">
            <widget class="QLineEdit" name="Velocidad_lineEdit"/>
           </item>
           <item row="3" column="3" colspan="2">
            <widget class="QLineEdit" name="DestinoY_lineEdit"/>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="Velocidad">
             <property name="text">
              <string>Velocidad:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3" colspan="2">
            <widget class="QLineEdit" name="Origen_X_lineEdit"/>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="Origen_Y">
             <property name="text">
              <string>Origen Y: (0 - 500)</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QLabel" name="Red">
             <property name="text">
              <string>Red: (0 - 255)</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QLabel" name="DestinoY">
             <property name="text">
              <string>Destino Y: (0 - 500) </string>
             </property>
            </widget>
           </item>
           <item row="14" column="3" colspan="2">
            <widget class="QLineEdit" name="Blue_lineEdit"/>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="Origen_X">
             <property name="text">
              <string>Origen X: (0 - 500)</string>
             </property>
            </widget>
           </item>
           <item row="16" column="1" colspan="4">
            <widget class="QPushButton" name="Agregar_inicio_pushButton">
             <property name="text">
              <string>Agregar inicio</string>
             </property>
            </widget>
           </item>
           <item row="17" column="1" colspan="4">
            <widget class="QPushButton" name="Agregar_final_pushButton">
             <property name="text">
              <string>Agregar Final</string>
             </property>
            </widget>
           </item>
           <item row="18" column="1" colspan="4">
            <widget class="QPushButton" name="Mostrar_pushButton">
             <property name="text">
              <string>Mostrar elementos</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPlainTextEdit" name="Salida"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tabla</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QLineEdit" name="buscar_lineEdit">
          <property name="placeholderText">
           <string>ID de la particula.</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="mostrar_tabla_pushButton">
          <property name="text">
           <string>Mostrar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="buscar_pushButton">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QTableWidget" name="tabla"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>615</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
